docs/tileentities/tesr.md

```diff
@@ -26,3 +26,29 @@ Registering a TER
 -----------------
 
 In order to register a TESR, call `ClientRegistry#bindTileEntitySpecialRenderer` passing the tile entity class to be renderer with this TER and the instance of the TER to use to render all TEs of this class.
+
+IRenderTypeBuffer - Vertex Builders
+-----------------------------------
+This class gives you access to `IRenderTypeBuffer`, which in turn provides a series of `IVertexBuilder` instances via the `IRenderTypeBuffer#getBuffer` method.
+
+To get an example vertex builder (VB), see the `RenderTypes` class for a set of prebuilt builders.
+
+There are various vertex builders that Minecraft provides by default; here is a small overview of the main VB instances:
+
+| Render Type |  Example Purpose
+| :---------- | :--------------
+| SOLID | Rendering block faces
```

Comment:
As these types are private, it would be better to refer to them by their getters (e.g. `#getSolid`, `#getCutoutMipped`, etc.)